ee/codegen/src/context/output-variable-context.ts

```diff
@@ -26,4 +36,26 @@ export class OutputVariableContext {
   public getRawName(): string {
     return this.outputVariableData.key;
   }
+
+  private generateSanitizedOutputVariableName(): string {
```

Comment:
Blindish copypasta of workflow-output-context. This is what clued me into the fact that we may remove that class once this clean up is done